.travis.yml

```diff
@@ -26,42 +26,82 @@
         "python": "3.3"
       },
       {
+        "env": "BUILD=test DAV_SERVER=xandikos REQUIREMENTS=devel BUILD_PRS=true ",
+        "python": "3.3"
+      },
+      {
         "env": "BUILD=test DAV_SERVER=radicale REQUIREMENTS=release BUILD_PRS=true ",
         "python": "3.3"
       },
       {
+        "env": "BUILD=test DAV_SERVER=xandikos REQUIREMENTS=release BUILD_PRS=true ",
+        "python": "3.3"
+      },
+      {
         "env": "BUILD=test DAV_SERVER=radicale REQUIREMENTS=minimal BUILD_PRS=true ",
         "python": "3.3"
       },
       {
+        "env": "BUILD=test DAV_SERVER=xandikos REQUIREMENTS=minimal BUILD_PRS=true ",
+        "python": "3.3"
```

Comment:
@jelmer Would you consider supporting Python 3.3+ (i.e. I submit PRs to xandikos if Python 3.3 breaks)